components/support/rc_crypto/nss/nss_build_common/src/lib.rs

```diff
@@ -141,6 +142,22 @@ fn get_nss_libs(kind: LinkingKind) -> Vec<&'static str> {
                     static_libs.push("intel-gcm-s_lib");
                 }
             }
+            // We need this due to using the NSS CI, which enables PKIX lib by default
+            // However this is only used for Macs testing android on desktop
+            // https://searchfox.org/mozilla-central/source/security/nss/lib/libpkix/libpkix.gyp#15-25
```

Comment:
I wonder if this should be a permalink? Or maybe just a link to the file itself without line numbers and a comment saying it should match the "dependencies" section would be ok?

Reply:
Ah good point! I chatted with the NSS team and I think we can actually just avoid having libpkix all together (disable it in the CI builds) but if it turns out we have to keep this around i'll definitely make those changes